src/components/Analytics/utils.ts

```diff
@@ -116,6 +117,8 @@ export function trackEvent(
       transport,
     });
 
+    gtag.event(action, category, label, value);
```

Comment:
Consider calling `toTitleCase` if needed (to match how events are being recorded for UA)